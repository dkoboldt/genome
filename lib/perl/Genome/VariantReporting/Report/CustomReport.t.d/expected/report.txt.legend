Headers
chromosome_name	Chromosome: an identifier from the reference genome or an angle-bracketed ID String ("<ID>") pointing to a contig in the assembly file
start	The start position of the variant. One-based
stop	The end position of the variant. One-based, inclusive
reference	The reference allele at this position
variant	The variant called at this position
ensembl_gene_id	Ensembl stable ID of affected gene
Filters
