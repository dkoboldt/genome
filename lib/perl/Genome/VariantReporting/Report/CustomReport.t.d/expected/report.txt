chromosome_name	start	stop	reference	variant	ensembl_gene_id
1	1	1	A	T	ENSG00000223659
1	1	1	A	G	ENSG00000223695
